content/efcore/getting-started/configuring-deletion.md

```diff
@@ -0,0 +1,60 @@
+# Configuring Deletion 
+ 
+When a record is deleted from a database, its dependent records are also affected. In this lesson, we will learn the various strategies a relational database can use to handle dependent records when a principal record is deleted, as well as how to configure each strategy in EF Core. 
+ 
+## Deletion Strategies 
+ 
+There are three strategies that a database can use to handle a dependent of a deleted record: 
```

Comment:
Instead of saying "strategy", lets use "behavior" everywhere. Don't just replace the word; make sure the sentence still flows and makes sense.